### ISSUE TITLE
PhpStorm - PHPUnit failed tests diff does not render content

### ISSUE THREAD

**Issue by rozsival** [ChrisRM/material-theme-jetbrains#1368]
### Summary

When running a test suite through IDE's `Run` tool, failing tests provide a nice side-to-side diff of expected and actual values. Newly opened window of the diff does not render any content when using Material Darker Theme.

### Expected behavior

Diff window should render content normally as when using any of IDE's default themes.

### Actual behavior

Material Darker themed diff window does not show any content and appears to be empty.

### Steps to reproduce

1. Create a unit test for [phpunit](https://github.com/sebastianbergmann/phpunit)
2. Open the test suite and implement any test case that will fail
3. Run the failing test using IDE's `Run` tool
4. Use `Click to see difference` in IDE's console output to open the diff window

Screencast: https://drive.google.com/open?id=1Crr5R2XFzVHaOAEFNhzfOENX0efGZwZ4

### Environment

* Plugin version: 4.5.1
* PhpStorm 2019.2.1 (Build #PS-192.6262.66, built on August 22, 2019)
* Runtime version: 11.0.3+12-b304.39 x86_64
* VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
* macOS 10.14.6
* GC: ParNew, ConcurrentMarkSweep
* Memory: 4062M
* Cores: 4
* Registry: run.processes.with.pty=TRUE, ide.mac.allowDarkWindowDecorations=true, ide.tree.ui.experimental=false, ide.balloon.shadow.size=0, javascript.flow.use.lsp=false
Non-Bundled Plugins: com.chrisrm.idea.MaterialThemeUI, com.intellij.ideolog, mobi.hsz.idea.gitignore, net.seesharpsoft.intellij.plugins.csv, com.dmarcotte.handlebars, NEON support, Nette framework helpers, com.jantvrdik.intellij.latte, intellij.prettierJS, org.psliwa.idea.composer, org.sylfra.idea.plugins.linessorter, ru.adelf.idea.dotenv

**Comment by rozsival** [comment]
@mallowigi Unfortunately, PhpStorm is not available in that version yet, not even through [EAP](https://www.jetbrains.com/phpstorm/eap/). The latest version 2019.2.2 does not solve the issue. I don't think it's critical, so maybe we can wait until PhpStorm is available in version 2019.3 and I will let you know then if the issue has been solved.

**Comment by mallowigi** [comment]
2019.3 is now available. Try with it.

**Comment by rozsival** [comment]
@mallowigi Tried it today, no change at all, still blank window. When I switch to any other theme, it works.

**Comment by mallowigi** [comment]
Can you post your environment now that you use the EAP?

**Comment by rozsival** [comment]
@mallowigi Sorry for late response. I tested it twice. First time, right after EAP 2019.3 installation, without any plugins or settings copied from the previous version, so fresh install, just with the Material Theme. It did not work. Then I set the very same environment up as posted in the issue. Same settings, projects, plugins. No change at all.

**Comment by rozsival** [comment]
I doubt it is PhpStorm issue, as I said, if I switch to any other theme (default by JetBrains or downloaded), it works without any issue.

**Comment by mallowigi** [comment]
Ok, I cannot reproduce. Can you disable the "Themed Title Bar" setting, restart and tries if this happen again ?

**Comment by rozsival** [comment]
@mallowigi I confirm turning off "Themed Title Bar" did solve the issue. Thanks!

What exactly this setting does? I see no difference between having it on and off 😄 
Also should we consider this a bug or we can close the issue as it's rather related to my setup?

**Comment by mallowigi** [comment]
TLDR: The Themed Title Bar is using an unfinished feature of JetBrains Runtime 11 which causes these blank windows. It's supposed to be fixed in the last version of the Runtime but if not, simply disable this feature.

These bugs about blank windows appear here and there, sometimes it's vcs, sometimes it's another windows... These are caused by a changed introduced in JBRuntime 11 when they re-introduced the transparent title bars.

The Material Theme always had this "Themed Title Bar" setting and it worked fine in JBR8, but they didn't introduced the change when they released the JBR11. So I asked them and instead of redoing the same implementation as in JBR8, instead they thought of completely removing the title bar, which was not asked and made many users complain to my plugin making use of that change. 

Since this wasn't a feature they asked for, they didn't bother fixing it, and instead someone helped me create a "fake title bar" to replace the missing title bar, by adding a component on the top of the screen acting like a real title bar. This fixed many of the complaints, however another issue, not related to my plugin but related to the "transparent title bar" feature of JBR11, was that sometimes, especially in full screen mode, windows stop rendering completely at first render, hence the blank windows. A workaround was found by resizing the said window to make the content reappear, but it's still "hacky"

Fortunately it seems like they fixed that behavior in the last JBR 11, at least for what I've seen, this is why I was surprised when you told me they didnt'. What I think is that you still have another version of the JBR, or worse, a custom JDK. 

Disabling the title bar reverts to the old behavior, e.g. the title bar being black or white instead of transparent.

At some point the transparent title bar feature will make its way into the main product and the plugin option will become obsolete. But for now it's not the case

**Comment by rozsival** [comment]
@mallowigi Understood! Thank you for your help and provided explanation!
